src/lib/sync.ts

```diff
@@ -25,10 +27,30 @@ export async function syncPrices(): Promise<{ synced: number }> {
 
 export async function syncSubscriptions(): Promise<{ synced: number }> {
   let synced = 0
-  for await (const subscription of stripe.subscriptions.list({ limit: 100 })) {
+  for await (const subscription of stripe.subscriptions.list({ status: 'all', limit: 100 })) {
     await upsertSubscription(subscription)
     synced++
   }
 
   return { synced }
 }
+
+export async function syncCustomers(): Promise<{ synced: number }> {
```

Comment:
just to check here: does this mean only the first 100 customers will be synced?

Reply:
stripe has very good [auto-pagination](https://stripe.com/docs/api/pagination/auto). That block of code will sync everything with 100 items per request

Reply:
oooh wow damn never knew that